AK/windows.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024, stasoid <stasoid@yahoo.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#ifdef AK_OS_WINDOWS // needed for Swift
```

Comment:
The "needed for swift" comment is confusing. Have you tried enabling swift on windows?

Reply:
If this is related to wanting to error out on other platforms if they include this header, the path to it will need updated in the ENABLE_SWIFT branch of AK/CMakeLists.txt to be an EXCLUDED path for the Swift bridging header. See LibCore for an example.

Reply:
I added #ifdef to fix [this error](https://github.com/LadybirdBrowser/ladybird/actions/runs/12104386695/job/33747734389?pr=2674). LibCore (or other libs) doesn't have the word EXCLUDED in their CMakeLists files. Can you provide the exact line that should be added to exclude Windows.h?

Reply:
Adding `set(SWIFT_EXCLUDE_HEADERS Windows.h)` to AK/CMakeLists.txt didn't fix the problem, so I am still using #ifdef.